modules/special-rules/pages/has-super-heavy-weapon.adoc

```diff
@@ -0,0 +1,36 @@
+= Has +{super-heavy weapon}+
+
+`Data sheet modifier`
+
+These units are specially adapted from their original design, to replace their main armament with a more specialised weapon.
+
+---
+
+[NOTE]
+.The +{super-heavy weapon}+ placeholder
+====
++{super-heavy weapon}+ is a placeholder that we use here to explain a number of related abilities.
+You won't see it in the army lists and unit data sheets -- rather, you'll see the name of a specific super-heavy weapon, as in {has-barrage} or {has-macro-weapon}.
+====
+
+Where your unit has the {has-super-heavy-weapon} ability, apply these changes relative to its base unit:
+
+* Replace its Firepower value with the corresponding super-heavy weapon.
+
+.The {has-death-ray} ability
+====
+The Mekboy Speedsta with Death Ray unit is a Mekboy Speedsta base unit with added {has-death-ray} ability.
+
+* The base unit has Firepower `1`, whilst the Mekboy Speedsta with Death Ray unit has a {death-ray} super-heavy weapon instead.
+====
+
+.The {has-macro-weapon} ability
+====
+The Leman Russ Vanquisher unit is a Leman Russ base unit with added {has-macro-weapon} ability.
+
+* The base unit has Firepower `3`, whilst the Leman Russ Vanquisher unit has a {macro-weapon} super-heavy weapon instead.
+====
+
```

Comment:
Do we really need two examples? They both demonstrate the exact same thing, just with different specific unit/weapon combinations. It strikes me as a bit redundant to have two examples when they aren't showing different variations in the application of a rule or process.

That said, I'm content to go with it if you feel strongly — it clutters things up a little more, but isn't a big deal.

Reply:
We don't _need_ two.

I have a minor preference for keeping both, as it shows the reader how it's being applied to make the original Epic 40k Mekboy Speedsta work, and to the 'new' Vanquisher, in a consistent manner. It also shows different numeric Firepower values being replaced with a single super-heavy weapon shot.

If you'd like to reduce it to one, I think the Vanquisher should be kept.